src/Model/CategoryManager.php

```diff
@@ -27,7 +27,7 @@ final class CategoryManager implements CategoryManagerInterface
      */
     private $categoryManager;
 
-    public function __construct(ManagerInterface $categoryManager)
+    public function __construct(?ManagerInterface $categoryManager = null)
```

Comment:
why it this change needed?

Reply:
Becouse `CollectionBundle` is optional/suggest. In functional test some warnings about it was displayed:
```xml
<service id="sonata.media.manager.category.default" class="Sonata\MediaBundle\Model\CategoryManager" public="true">
    <argument type="service" id="sonata.classification.manager.category" on-invalid="null"/>
</service>
```